apps/site/assets/ts/models/__tests__/alert-test.ts

```diff
@@ -398,7 +398,7 @@ describe("alertsAffectingBothDirections", () => {
   });
 });
 
-describe("hasFacilityAlert", () => {
+describe("hasCurrentFacilityAlert", () => {
```

Comment:
nit-pick: could there be a test to ensure that non current alerts are filtered out from the `hasCurrentFacilityAlert` function.